taca/server_status/cronjobs.py

```diff
@@ -9,35 +9,34 @@
 
 from taca.utils.config import CONFIG
 
-def update_cronjob_db():
-    logging.info('Getting list of cronjobs')
-    crontab = CronTab()
-    # CronTab() is supposed to return the list of cronjobs
-    # it works on milou, but doesn't work on preproc (returns an empty list)
-    # when it doesn't work, create CronTab object from the output of command 'crontab -l'
-    if crontab.crons == []:
-        try:
-            p = subprocess.Popen('crontab -l', shell=True, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
-        except Exception, e:
-            logging.error(e.message)
-        else:
-            output = p.communicate()[0]
-            crontab = CronTab(tab=output)
+def _parse_crontab():
+    result = {}
+    user = getpass.getuser()
+    logging.info('Getting crontab for user {}'.format(user))
+    try:
+        crontab = CronTab(user=user)
```

Comment:
Have tried it in any of the `NAS` ? Cause when I tried (out of curiosity) I got the following error 👀 

```
>>> crontab = CronTab(user='hiseq.bioinfo')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/hiseq.bioinfo/opt/anaconda/envs/master/lib/python2.7/site-packages/crontab.py", line 202, in __init__
    self.read(tabfile)
  File "/home/hiseq.bioinfo/opt/anaconda/envs/master/lib/python2.7/site-packages/crontab.py", line 237, in read
    raise IOError("Read crontab %s: %s" % (self.user, err))
IOError: Read crontab hiseq.bioinfo: must be privileged to use -u
```

Anyway it is not something related to the code itself, maybe we should ask `IT` to fix it (i.e. if it is doable) ?